os/arch/arm/src/s5j/s5j_spi.c

```diff
@@ -557,7 +560,7 @@ struct spi_dev_s *up_spiinitialize(int port)
 
 	/* SET SPI INITIAL */
 	SPI_SFR *pSPIRegs;
-	pSPIRegs = (SPI_SFR *)priv->base;
+	pSPIRegs = (SPI_SFR *) priv->base;
```

Comment:
All cases in this file are same.
remove blank after type casting..

Reply:
done